seahorn/sea.yaml

```diff
@@ -34,12 +34,14 @@ verify_options:
   horn-bv2-lambdas: ''
 # use z3 simplifier during vcgen
   horn-bv2-simplify: true
-# wide memory manager to track pointer sizes
```

Comment:
do not expect this PR to change formatting on this file

Reply:
Fixed.